Banno/jha-design
# chore(deps): replace dependencies that use vulnerable lodash.template

## Summary

This removes the use of `lodash.template` by replacing the following dependencies:

* Replaces `gulp-cssmin` with `gulp-cleancss`
   - The resulting CSS in _dist/css/style.min.css_ is slightly different, but it seems to simply involve reorganization of the CSS
* Replaces `gulp-header` with `gulp-prepend`

Addresses https://github.com/Banno/jha-design/security/dependabot/79

## How to test
1. Check out master and run a dev server from the `dist/` directory, such as `npx http-server -p 3000`
2. Load the web page
3. Check out this branch, repeat steps 1 and 2, and compare the CSS
